test/unit/Reflection/ReflectionFunctionAbstractTest.php

```diff
@@ -745,6 +745,32 @@ function foo() {}', $comment);
         self::assertTrue($function->isDeprecated());
     }
 
+    /**
+     * Set $parser = null in case the error below occur when excute with phpunit's data provider
+     * Error : Typed static property Roave\BetterReflection\Reflection\ReflectionFunctionAbstract::$parser must not be accessed before initialization
+     *
+     * @dataProvider getDataForTypedAccessTest
+     */
+    public function testAccessTypedConstantWithDataProvider(array $data): void
+    {
+        $x = ReflectionFunctionAbstract::CLOSURE_NAME;
+
+        $this->assertEquals('{closure}', $x);
```

Comment:
This is verifying a constant value is equal something, and it does not use `$data`? :thinking: 

Is this a WIP test?

Reply:
it's nothing about the constant or the $data, run this test case with the original `private static ?Parser $parser;` will get error, but `private static ?Parser $parser = null;` not.

Reply:
So the triggering factor is `ReflectionFunctionAbstract::CLOSURE_NAME;`?

Asking because I'm mostly trying to reduce this further into a smaller test case, rather than a side-effect of code floating within the test suite.